talking_stick.gemspec

```diff
@@ -17,8 +17,8 @@ Gem::Specification.new do |s|
   s.files         = `git ls-files`.split("\n")
   s.test_files    = `git ls-files -- {test,spec,features}/*`.split("\n")
 
-  s.add_dependency 'rails', '> 3.0', '< 5.0'
-  s.add_dependency 'jquery-rails', '> 3.0', '< 5.0'
```

Comment:
Since this is a breaking change we'll need to increment the major number. I suggest making one final release of TalkingStick (compatible with Rails 3 & 4) prior to merging this.

Reply:
Or we could make it backward compatible fairly trivially.

Reply:
@etagwerker Do you think we can make this backward compatible? I think the only thing we need to wrap in a conditional is a shim for `before_filter`, and then loosen up the jquery-rails dependency.

Reply:
@benlangfeld I don't think it's that simple to do that. There is also migration code that is not backward compatible: 

```
# for example
class CreateTalkingStickRooms < ActiveRecord::Migration[5.1]
```

We would have to add a shim there as well considering these versions: `["4.2", "5.0", "5.1"]`

Is it really worth it though? 

I'd suggest we create a `stable-4` branch with what's in `develop`/`master` right now and we release one last version of `talking_stick`.

Reply:
@etagwerker It is not necessary to maintain compatibility with Rails < 5.0.  Anyone on older Rails that wants to use TalkingStick can use the older version.

Reply:
@etagwerker Which migrations use Rails 5 features? I was under the impression that a migration targeting 4.2 would also execute on 5.

Reply:
@benlangfeld Good point. I guess we can make all migrations extend from `ActiveRecord::Migration[4.2]` and define the dependency like this: 

```
s.add_dependency 'rails', '>= 4.2', '< 6.0'
```

I can work on the shim for `before_filter` | `before_action`